external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -2158,6 +2158,36 @@ abstract class AvroSuite
       }
     }
   }
+
+  test("SPARK-33865: Hive DDL with avro should check col name") {
```

Comment:
why do we mention Hive? The test uses `USING AVRO` which is Spark native CREATE TABLE DDL.

Reply:
Done